### ISSUE TITLE
Invalid link

### ISSUE THREAD

**Issue by dentiny** [chenjiandongx/mandodb#4]
Hi, I think the link to "Write a time-series database engine from scratch", might be this one? https://nakabonne.dev/posts/write-tsdb-from-scratch/

**Comment by huang-jesse** [comment]
No, the original article has invalid, there is the archieve [here](https://web.archive.org/web/20190220210543/https://fabxc.org/tsdb/)
